README.md

```diff
@@ -182,7 +182,7 @@ You can write SpacetimeDB modules in a bunch of popular languages, with many mor
 - [Rust](https://spacetimedb.com/docs/sdks/rust/quickstart)
 - [C#](https://spacetimedb.com/docs/sdks/c-sharp/quickstart)
 - [Typescript](https://spacetimedb.com/docs/sdks/typescript/quickstart)
-- [Python](https://spacetimedb.com/docs/sdks/python/quickstart)
+- Python (planned)
```

Comment:
good catch.. I wonder what I searched for / why I missed this..

Reply:
I think you correctly made the change to the docs, but it's also listed here.

Reply:
oh yeah this is a different repo :upside_down_face: